openstack/nova/values.yaml

```diff
@@ -197,6 +197,7 @@ compute:
       max_concurrent_builds_per_project: 3
       ram_allocation_ratio: 1.0
       cpu_allocation_ratio: 8.0
+      block_device_allocate_retries: 300
```

Comment:
That's not enough. You'll have to make (the template)[https://github.com/sapcc/helm-charts/blob/master/openstack/nova/templates/hypervisors/hypervisors-vmware-configmap-vct.yaml#L21-L35] use that variable.

Reply:
Ok, done

Reply:
Maybe you could comment here, that this equals 15min. Other than that I'm fine with it.